.github/ISSUE_TEMPLATE/01-github-request---onboarding.yml

```diff
@@ -19,10 +19,10 @@ body:
   - type: textarea
     id: users
     attributes:
-      label: User List (case sensitive!) - Please also include in ticket title
-      description: Which user(s) are you requesting access for? Note GitHub usernames are case-sensitive, so please enter in the correct capitalization.
+      label: User List (case sensitive!) - Please also include usernames in ticket title
+      description: Who are you requesting access for, and what are their GH usernames and emails? GH usernames are case-sensitive, so please enter in the correct capitalization.
```

Comment:
Should we indicate a way for them to PM us the email addresses, in case they don't want them fully visible on the interwebs?

Reply:
Also, emails aren't required for all onboarding requests, right? Only 2U onboards people to their CLA through this form. People asking to onboard a user to say, `openedx-triage` we don't need email addresses. I wonder if we should have 2U follow the process everyone else follows to get them attached to their entity CLA.

Reply:
Per discussion in standup, we'll close this PR and work towards 2U using the same process as other firms with entity CLAs.